raspberry/read_sbus/read_sbus_from_GPIO_receiver.py

```diff
@@ -1,46 +1,92 @@
 import read_sbus_from_GPIO
 import time
 import serial
+from radioDataParser import RadioDataParser
+import threading
+
 
 SBUS_PIN = 25 #pin where sbus wire is plugged in
-ser = serial.Serial('/dev/ttyAMA0', 115200, timeout=1) #serial connection between Pi and ESP32
+ser = serial.Serial('/dev/ttyUSB0', 500000, timeout=1) #serial connection between Pi and ESP32
 ser.reset_input_buffer()                               #serial.Serial(port, buad rate, timeout)
 
 reader = read_sbus_from_GPIO.SbusReader(SBUS_PIN)
 reader.begin_listen()
 
 #wait until connection is established
 while(not reader.is_connected()):
-    time.sleep(.2)
+        time.sleep(.2)
 
 #Note that there will be nonsense data for the first 10ms or so of connection
 #until the first packet comes in.
 time.sleep(.1)
 
-while True:
+parser = RadioDataParser()
+
+# TODO: implement the below loop as a thread with frequency 50Hz. Might want to .join() and manage threads using 
```

Comment:
Before merging, revise these comments. Has this "todo" been completed?

Reply:
This `TODO` I want to keep for future reference as we keep adding threads, we might face sync issues and racing conditions. In those comments, I have included a workaround for resolving these issues. It won't be obvious when these issues occur as they don't throw any errors or exceptions, so I thought to keep them there for future reference.